src/common/framework/fixture.ts

```diff
@@ -166,6 +166,13 @@ export class Fixture<S extends SubcaseBatchState = SubcaseBatchState> {
     throw new SkipTestCase(msg);
   }
 
+  /** Throws an exception marking the subcase as skipped if condition is true */
```

Comment:
Nice! I'm surprised we didn't already have this... Also probably tons of places could be eventually updated to use this!